docs/src/components/Header.js

```diff
@@ -77,16 +77,16 @@ export default function Header({ onChangeColorScheme }: Props) {
             <Tooltip inline text="Right-To-Left View">
               <IconButton
                 size="md"
-                accessibilityLabel="toggle page direction"
+                accessibilityLabel="toggle page direction: Left-To-Right / Right-To-Left View"
                 iconColor="white"
                 dangerouslySetSvgPath={togglePageDirSvgPath}
                 onClick={toggleRTL}
               />
             </Tooltip>
-            <Tooltip inline text="Toggle color scheme">
+            <Tooltip inline text="Dark-Mode View">
```

Comment:
This is a little odd since it says "Dark-Mode View" which ever color scheme you're currently in. What about "Toggle dark/light mode"?